src/__tests__/ActionList.test.tsx

```diff
@@ -155,6 +155,21 @@ describe('ActionList', () => {
     expect(option).toBeInTheDocument()
   })
 
+  it('should call onClick for a link item', async () => {
+    const onClick = jest.fn
+    const component = HTMLRender(
+      <ActionList role="listbox">
+        <ActionList.LinkItem onClick={onClick}>Primer React</ActionList.LinkItem>
+      </ActionList>,
+    )
+    const link = await waitFor(() => component.getByRole('link'))
```

Comment:
Overall, this looks great, but I don't think this test is passing based on this line. It doesn't look like the 'link' role is assigned at all. If you can button that up, I think this is good to go.

Thanks for knocking this out!